### PR TITLE
page explaining how to set custom error/maintenance page

### LINKED ISSUE #300: There should be a page explaining how to set custom error/maintenance page

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #300

### CODE REVIEW

**Comment by EtienneM** [comment]
I still have a question regarding the `SCALINGO_NO_FRONT_ERROR_URL` env variable (https://github.com/Scalingo/documentation/pull/302/files#diff-a5fa1443d61657f228f1fba02ac6730dR10)

File: _posts/platform/app/2000-01-01-custom-error-page.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Custom error page
+modified_at: 2018-03-22 00:00:00
+tags: app custom error page
+---
+
+Our router may return an error page in various cases, when an application is unreachable:
+
+- The amount of container for the application is 0.
+- TODO no front == no web containers?!
```
Comment (Soulou) [inline _posts/platform/app/2000-01-01-custom-error-page.md]: Yes web containers scaled to 0 while other containers are running

File: _posts/platform/app/2000-01-01-custom-error-page.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Custom error page
+modified_at: 2018-03-22 00:00:00
+tags: app custom error page
+---
+
+Our router may return an error page in various cases, when an application is unreachable:
```
Comment (Soulou) [inline _posts/platform/app/2000-01-01-custom-error-page.md]: router**s**?

File: _posts/platform/app/2000-01-01-custom-error-page.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+- The amount of container for the application is 0.
+- TODO no front == no web containers?!
+- The application returns a 5XX error.
```
Comment (Soulou) [inline _posts/platform/app/2000-01-01-custom-error-page.md]: That's wrong actually, 5XX errors should be those if the application

Comment (Soulou) [reply _posts/platform/app/2000-01-01-custom-error-page.md]: It's when the application crashes of cut the connection abrubptely (our router returns 502 + error page)

File: _posts/platform/app/2000-01-01-custom-error-page.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+- `SCALINGO_STOPPED_PAGE_URL`
+- `SCALINGO_NO_FRONT_ERROR_URL`
+- `SCALINGO_APP_ERROR_URL`
+- `SCALINGO_TIMEOUT_ERROR_URL`
+
+The usage is the following:
+`SCALINGO_STOPPED_PAGE_URL="https://cdn.example.com/stopped_page.html"`.
+
+In order to prevent the page to be fetched at each request, we cache its
```
Comment (Soulou) [inline _posts/platform/app/2000-01-01-custom-error-page.md]: The content is cached

File: _posts/platform/app/2000-01-01-custom-error-page.md (hunk 3 above)
Comment (Soulou) [reply _posts/platform/app/2000-01-01-custom-error-page.md]: No I mean if an app return 500, it's forwarded, it's not our error page

File: _posts/platform/app/2000-01-01-custom-error-page.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+title: Custom error page
+modified_at: 2018-03-22 00:00:00
+tags: app custom error page
+---
+
+Our router may return an error page in various cases, when an application is unreachable:
+
+- The amount of container for the application is 0.
```
Comment (Soulou) [inline _posts/platform/app/2000-01-01-custom-error-page.md]: - There is no running container

?

File: _posts/platform/app/2000-01-01-custom-error-page.md [hunk 6]
```diff
@@ -4,11 +4,11 @@ modified_at: 2018-03-22 00:00:00
...
 ---
 
-Our router may return an error page in various cases, when an application is unreachable:
+Our routers may return an error page in various cases, when an application is unreachable:
 
-- The amount of container for the application is 0.
-- TODO no front == no web containers?!
-- The application returns a 5XX error.
+- There is no running container.
+- The application has no web container running but other type of containers are.
```
Comment (Soulou) [inline _posts/platform/app/2000-01-01-custom-error-page.md]: other type**s** ?

**Comment by Soulou** [comment]
There should be something to speak about "Maintenance screen" like "you might want to set these variables to have a custom maintenance page as it will display when you'll stop your app for maintenance operations"

**Comment by yannski** [comment]
@EtienneM can you complete the article around the 2 sections "Error Pages" / "Maintenance Pages"?

**Comment by EtienneM** [comment]
Yep will do it in a couple of minutes. Wanted to open all the PR for the router logs first.

**Pull request merged**